pep-0604.rst

```diff
@@ -74,61 +76,72 @@ Simplified Syntax
     assert str | int == Union[str,int]
     assert str | int | float == Union[str, int, float]
```

Comment:
I think the call example and the asserts are unnecessary (what the asserts say is repeated better below).